examples/dynamics/absolutely inelastic collision.py

```diff
@@ -0,0 +1,46 @@
+#inelastic collision transforms two objects with masses m1, m2 and velocities v1 and v2 to one object with mass m = m1+m2 and unknown velocity to be resolved
```

Comment:
File name should not contain spaces.
Rename to `absolutely_inelastic_collision.py`

Reply:
Not fixed.